test/sqllogictest/transform/fold_vs_dataflow/3_number_aggfns_dataflow.slt

```diff
@@ -55,7 +55,7 @@ FROM t_using_dataflow_rendering;
 Explained Query:
   Return
     Project (#0..=#17, #30..=#35)
-      Map ((#18 / bigint_to_real(case when (#19 = 0) then null else #19 end)), (#20 / bigint_to_double(case when (#21 = 0) then null else #21 end)), (numeric_to_double(#22) / bigint_to_double(case when (#23 = 0) then null else #23 end)), (#24 / bigint_to_real(case when (#25 = 0) then null else #25 end)), (#26 / bigint_to_double(case when (#27 = 0) then null else #27 end)), (numeric_to_double(#28) / bigint_to_double(case when (#29 = 0) then null else #29 end)))
+      Map ((#18 / bigint_to_real(case when (#19 = 0) then null else #19 end)), (#20 / bigint_to_double(case when (#21 = 0) then null else #21 end)), (#22 / bigint_to_numeric(case when (#23 = 0) then null else #23 end)), (#24 / bigint_to_real(case when (#25 = 0) then null else #25 end)), (#26 / bigint_to_double(case when (#27 = 0) then null else #27 end)), (#28 / bigint_to_numeric(case when (#29 = 0) then null else #29 end)))
```

Comment:
I think this is this change: https://github.com/MaterializeInc/materialize/pull/15612 Can you confirm, @sploiselle?  

(Context: this is a test that has not been running in CI for a long time, and we are trying to make it run now, but some things have changed, so we are doing some archaeology to figure out if these changes are ok.)